ui/src/app/components/sidenav-view-assessment/sidenav-view-assessment.component.scss

```diff
@@ -0,0 +1,82 @@
+.header {
```

Comment:
Please place these css in styles.css as much as possible. We should avoid local css.

Reply:
Updated